roles/nginxplus/vars/main.yml

```diff
@@ -1,2 +1,64 @@
 ---
 # vars file for roles/nginxplus
+nginx_signed_statement: "[signed-by=/usr/share/keyrings/nginx-archive-keyring.gpg]"
```

Comment:
How did you decide what should go here and what should go into `group_vars/common.yml`?